README.md

```diff
@@ -174,5 +176,5 @@ If running the latest stable version of Shellcheck is not to your liking, you ca
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      with:
-       version: v0.7.0
+       version: v0.9.0
```

Comment:
Neither of those versions seem available as [github releases](https://github.com/ludeeus/action-shellcheck/releases)

Reply:
How about 1.0.0?
since v1.1.0 is latest, that would be the next viable candidate

Reply:
I chose this per the README
> When specifying a custom version, please use any of the released versions listed in the [Shellcheck repository](https://github.com/koalaman/shellcheck/tags).
The version is presumably for `shellcheck`, not for `action-shellcheck`

Reply:
This is indeed for the binary, but its also not related to this.